pontoon/base/models.py

```diff
@@ -1525,7 +1524,9 @@ def available_locales_list(self):
 
 class UserProfile(models.Model):
     # This field is required.
-    user = models.OneToOneField(User, models.CASCADE, related_name="profile")
+    user = models.OneToOneField(
+        User, models.CASCADE, related_name="profile", primary_key=True
```

Comment:
`primary_key=True` is required to make Django to figure out the relation between the objects and remove the IntegrittyError